scripts/compile_solvers.sh

```diff
@@ -15,6 +15,7 @@ export K_AUG_BRANCH="ma57"
 export K_AUG_REPO="https://github.com/dthierry/k_aug"
 
 # Work-around for mumps gcc v10 gfortran bug
+# Some reason this did not work on macOS. That is b/c gcc points to the clang/XCode version.
 export GCC_VERSION=`gcc -dumpversion`
 if [ "$(expr substr "$GCC_VERSION" 1 2)" = "10" ]; then
```

Comment:
I had to manually add this to the coinbrew line. Help on debugging this is appreciated. It would be nice to automate.

Reply:
> 
> 
> @eslickj I added a bunch of comments to highlight where we could use your help.

@adowling2, sorry I totally missed these questions.  Let me take a closer look at this.  I guess we need some config file to maybe set some of these options.  The scripts currently aren't very flexible since I just have some very specific build environments in docker containers.

Reply:
Here maybe we could replace gcc by a variable that we would set to gcc.  In the build, are you using clang or gcc?  I guess we could also add a test to see if it's clang then if it's gcc, check the version.